app.js

```diff
@@ -25,15 +26,17 @@ app.get('/', function get(req, res) {
 });
 
 app.get('/responses', function get(req, res) {
-return smartSurveyAPICall.getData(
-    config.apiToken,
-    config.apiTokenSecret,
-    config.surveyID)
-  .then(results => res.json(results))
-  // This is a placeholder for error logging. In the future, the aim
-  // is to print a friendly message to the user with the stacktrace
-  // eslint-disable-next-line no-console
-  .catch(error => console.log(error));
+  const options = {
```

Comment:
This is probably just me, but this code is unpleasant to test. The only way to stub config is to use `proxy-require` and friends. I suggest you consider another approach and pass in the config as an argument. In this particular case I would refactor the code as:

```js
function createGet(config) {
  return get(req, res) {/* here you have your get route */}
}

app.get('/responses', createGet({token, tokenSecret, surveyId}));
```

Now testing becomes much easier since the function doesn't take any parameter from the outside and 
@daniel-ac-martin is ecstatic because the function is referentially transparent.